src/styles/index.scss

```diff
@@ -1,3 +1,4 @@
-body {
-  margin: 0;
-}
+@import './utils/variables';
```

Comment:
Ensure that the './utils/variables' file exists and contains definitions for all the variables used in the imported SCSS files, such as `$white-color`, `$main-color`, `$secondary-color`, and `$blue-color`. Missing variable definitions will cause compilation errors.